### ISSUE TITLE
HTTP 500 after browser back from login page

### ISSUE THREAD

**Issue by DeGuitard** [djyde/cusdis#10]
### Steps to reproduce:
1. Go to cusdis.com
2. Click on "Try it now"
3. Click on browser back button

### Expected result:
We are back to cusdis.com

### Actual result:
![image](https://user-images.githubusercontent.com/5683287/115504646-e1f9bf80-a295-11eb-8b9a-52a95cd678ad.png)

### Trace
Using the docker-compose file given in the documentations, I could reproduce the error. The container cusdis terminates with the following error:
```
cusdis_1  | /app/node_modules/next-auth/dist/client/index.js:181
cusdis_1  |       callbackUrl = window.location,
cusdis_1  |                     ^
cusdis_1  | ReferenceError: window is not defined
cusdis_1  |     at /app/node_modules/next-auth/dist/client/index.js:181:21
cusdis_1  |     at Generator.next (<anonymous>)
cusdis_1  |     at asyncGeneratorStep (/app/node_modules/next-auth/dist/client/index.js:33:103)
cusdis_1  |     at _next (/app/node_modules/next-auth/dist/client/index.js:35:194)
cusdis_1  |     at /app/node_modules/next-auth/dist/client/index.js:35:364
cusdis_1  |     at new Promise (<anonymous>)
cusdis_1  |     at /app/node_modules/next-auth/dist/client/index.js:35:97
cusdis_1  |     at _signIn (/app/node_modules/next-auth/dist/client/index.js:239:18)
cusdis_1  |     at signIn (/app/node_modules/next-auth/dist/client/index.js:173:18)
cusdis_1  |     at Dashboard (/app/.next/server/pages/dashboard.js:322:68)
```

### Versions
Cusdis image: djyde/cusdis:latest
Browser: Firefox 87.0 and Chrome 89.0.4389.128

**Comment by djyde** [comment]
Is it in "signed in" status or "not signed in" status?

**Comment by DeGuitard** [comment]
In a “not logged in” status. The browser back button is hit from the login page itself.

**Comment by egoist** [comment]
that should be fixed with my pr #7

**Comment by djyde** [comment]
@egoist yes your PR fixed this
